controllers/storagecluster/external_resources.go

```diff
@@ -710,6 +710,38 @@ func getTopologyConstrainedPoolsExternalMode(data map[string]string) (string, er
 	return string(topologyConstrainedPoolsStr), nil
 }
 
+// getFailureDomainKeyFromTopologyLabel converts short topology failure domain names to full Kubernetes label names
+func getFailureDomainKeyFromTopologyLabel(topologyLabel string) string {
```

Comment:
instead of reinventing this one can we refractor `getFailureDomainKeyFromStorageClassParameter` to work for both?